openwisp_monitoring/urls.py

```diff
@@ -1,5 +1,11 @@
 from django.urls import include, path
 
 urlpatterns = [
-    path('', include('openwisp_monitoring.device.api.urls', namespace='monitoring'))
+    path(
+        '',
+        include('openwisp_monitoring.device.api.urls', namespace='device_monitoring'),
```

Comment:
I have changed the namespace here.  Is this okay? Or, should we give a different namespace to the other API?

Reply:
other API for backward compatibility, we are 1.0.0 now, keep in mind